website/docs/components/dropdown/partials/code/component-api.md

```diff
@@ -75,10 +75,14 @@ The Dropdown component is composed of different child components each with their
   </C.Property>
   <C.Property @name="width" @type="string" @valueNote="any valid CSS width (px, rem, etc)">
     By default, the Dropdown List has a `min-width` of `200px` and a `max-width` of `400px`, so it adapts to the content size. If a `@width` parameter is provided then the list will have a fixed width.
+    <br/><br/>We discourage the use of percentage values for this argument. The Dropdown list is [a `popover` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover), so relative units use `document` as a reference (not the parent element), meaning percentage values act similar to `vw`.
```

Comment:
[praise] nice! 👌

Reply:
[question] should we specify here (and potentially in the "how to use" example) that if a consumer sets `@matchToggleWidth={{true}}` the `@width` argument is ~~ignored~~ irrelevant?